sr_unix/rtnhdr.h

```diff
@@ -237,7 +240,7 @@ typedef struct
 	 * references to this routine header. If not, it is ripe for cleanup as it has been replaced so drive zr_unlink_rtn()	\
 	 * on it.														\
 	 */															\
-	if ((RTNHDR)->rtn_relinked)												\
+	if ((NULL != (RTNHDR)) && (RTNHDR)->rtn_relinked)									\
```

Comment:
How is a null routine header address in a base frame possible? I thought we always fill it with some non-NULL base address even if not completely setup yet? Is this a test that failed? Just for my understanding.
    - sr_unix/rtnhdr.h
      1. Modify CLEANUP_COPIED_RECURSIVE_RTN() macro such that a null routine header address in a
         base frame doesn't screw things up and cause that NULL to be dereferenced.

Reply:
Previously, the call-in base frame had the program address of GTM$CI set. Now there's nothing to put there. I thought about putting GTM$DMOD there but (a) I might have to build it when it's not otherwise necessary, (b) that could be confusing since it really doesn't below there and (c) there were just a couple places that needed this check to keep from going bang if the routine hdr addr field was NULL.

Reply:
So do we now have a NULL rtnhdr for a call-in base frame? If so, adding a comment that (NULL != RTNHDR) is needed for a call-in base frame would help.

Reply:
Comment added.